sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveInspectors.scala

```diff
@@ -1063,6 +1063,9 @@ private[hive] trait HiveInspectors {
       case DateType => dateTypeInfo
       case TimestampType => timestampTypeInfo
       case NullType => voidTypeInfo
+      case dt =>
+        throw new AnalysisException("HiveInspectors does not support convert " +
```

Comment:
nit: `s"${dt.catalogString}" cannot be converted to Hive TypeInfo"`

Reply:
> nit: `s"${dt.catalogString}" cannot be converted to Hive TypeInfo"`

same reason like https://github.com/apache/spark/pull/29085#discussion_r459189448